openpower/package/ppe42-binutils/Config.in

```diff
@@ -0,0 +1,6 @@
+config BR2_PACKAGE_HOST_PPE42_BINUTILS
+        bool "ppe42-binutils"
+        default y if (BR2_OPENPOWER_PLATFORM && BR2_powerpc_power9)
```

Comment:
There is no such thing as BR2_powerpc_power9. Even if there was, you certainly don't need to depend on it